File: 02_classes_metodos/ruby/ex02.rb
require_relative 'ex01'

class Funcionario < Pessoa
  def initialize(cargo, nome, idade)
    super(nome, idade)
    @cargo = cargo

  end

  def trabalhar
    puts "#{@nome} esta trabalhando como #{@cargo}"
  end
end

pessoa = Funcionario.new("Engenheiro", "Deivid", 37)
pessoa.trabalhar

File: 02_classes_metodos/python/ex02.py
from ex01 import Pessoa

class Funcionario(Pessoa):
    def __init__(self, nome, idade, cargo):
        super().__init__(nome, idade)
        self.cargo = cargo

    def trabalhar(self):
        print(f"{self.nome} esta trabalhando como {self.cargo}.")


if __name__ == "__main__":

    funcionario = Funcionario("Deivid", 36, "Engenheiro")
    funcionario.trabalhar()